### DISCUSSION TITLE
No me ha llegado la invitación a backend

### DISCUSSION THREAD

**Discussion by Solymandias** [Launch-X-Latam/MisionFrontEnd#490] [Team Posts]
No me llegó al correo de launchX ni al correo que use en github

**Comment by JessicaSomar98** [comment]
A mí tampoco, ya revisé en el spam también y nada

**Comment by CharlieCCR** [comment]
No se preocupen, les va a llegar tarde o temprano, ya han aclarado que aun se siguen enviando las invitaciones correspondientes, igual cualquier problema me parece que hay un correo para pedir ayuda, o sino desde el propio teams Frontend hay gente preguntando la misma situación

**Comment by MAXP50** [comment]
cheken en todos sus bandejas de  correos  del 5 de noviembre

**Comment by SilvyeCG** [comment]
Mismo caso :( en el link donde nos registramos me aparece que ya estoy registrada pero no tengo la invitación a teams en ningun lado...me registre desde que enviaron el link pero no me llego ni me ha llegado :'(

**Comment by jriveramdev** [comment]
@JessicaSomar98 @SilvyeCG revisen su correo, en mi caso me llegó el 5 de noviembre, seguí las instrucciones pero me mostraba que ya estaba en ambos equipos de launchX sin embargo en teams solo estaba en FrontEnd, acabo de ver de nuevo el correo y ahí viene el link para solicitar unirse a teams backend, lo acabo de hacer y ya me aceptaron. Revisen de nuevo y para solicitar unirse es el primer link.

**Comment by leandrho** [comment]
Igual, a mi tampoco me llego, acabo de revisar los correos y no hay nada. Solo puedo acceder al mision-backend de github. :(

**Comment by EdgarToledano** [comment]
Llega al correo personal que registraron al momento de registrarse en el programa de launchx

**Comment by SilvyeCG** [comment]
Yo acabo de recibir justo hoy el correo para unirme a teams, tal vez es que no han terminado de mandar las invitaciones
